gendilsig.c

```diff
@@ -173,13 +179,16 @@ int main(int argc, char** argv)
 
     if(0 == sRc)
     {
+        unsigned char rngBuffer[32];
+        simple_rng(rngBuffer, sizeof(rngBuffer));
         printf("Generating Dilthium R2 8x7 signature ...\n");
         int gRc = mlca_sign(sSignature,
                             sSignatureBytes, /// validate RC
                             sDigest,
                             sDigestBytes,
                             sPrivKey,
                             sPrivKeyBytes,
+                            rngBuffer,
```

Comment:
Interesting in my review for this I just passed NULL as I think that will cause it to use rand under the covers as well.

Reply:
Updated